resources/public/api/conf/1.0/application.yaml

```diff
@@ -592,37 +910,69 @@ components:
         - dateTime
         - message
         - debugMessage
+      properties:
+        dateTime:
+          $ref: '#/components/schemas/DateTime'
+        message:
+          type: string
+        debugMessage:
+          type: string
+
+    DownstreamError:
```

Comment:
I wonder if we could just document this as one error structure but mark the additional fields we get back from EIS as optional then both errors could just be represented with one type. This might make the "making it more consistent" job simpler as it would then just have to update one type and we would be able to get rid of the "oneOf" instances that use this or the other response?